playwright/e2e/suites/test.spec.js

```diff
@@ -15,67 +15,69 @@ test.beforeEach(async ({ context }, testInfo) => {
   // so we don't have to depend on a Keycloak server to run the test suite. The
   // session value here is the base64-encoded session ID from dump/session.sql.
   await context.addCookies([
-    { name: 'SESSION', value: 'Mjc2NzE0YWQtYWIwYS00OGFhLThlZjgtZGI2NWVjMmU5NTBh', url: 'http://127.0.0.1:3000' },
+    { name: 'SESSION', value: 'YTVhOGIzODQtNDRiMi00NjIzLWIwNzgtZjk5NzI1NDFjOWU4', url: 'https://staging.terraware.io' },
   ]);
 });
 
 test('Add A Species', async ({ page }, testInfo) => {
-  await page.goto('http://127.0.0.1:3000');
+  await page.setViewportSize({ width: 1600, height: 1200 });
+
+  await page.goto('https://staging.terraware.io');
   await waitFor(page, '#home');
 
-  await page.waitForTimeout(5000);
+  const newSpeciesName = `Acacia koa-${new Date().getTime()}`;
+
   await page.getByRole('button', { name: 'Species' }).click();
-  await page.waitForTimeout(5000);
-  await page.getByRole('button', { name: 'Add Species' }).click();
-  await page.waitForTimeout(5000);
+
+  while (!(await page.locator('.dialog-box--opened').isVisible())) {
+    await sleep(1000);
```

Comment:
Why do we need this?  The default timeout on the locator should already handle waiting right?

Reply:
I added this because the container seemed to be rerendering after the button is clicked immediately. This is an indication of a rerender breaking the test flow, so this just makes sure that the dialog gets opened if it gets closed from the rerender

Reply:
Same story with the `while` checking for the `.options-container` below